examples/lib/defaultHtmlStepUi.js

```diff
@@ -272,7 +282,7 @@ function DefaultHtmlStepUi(_sequencer, options) {
         step.linkElements[index].href = step.imgElement.src;
     }
 
-    // TODO: use a generalized version of this
+    // TODO: use a generalized version of this.
```

Comment:
this means?

Reply:
IDK. Whoever added the TODO must know.

Reply:
I did! I meant that we might use a standard lib to determine file extensions, rather then a one-off function I made up 😄